### PR TITLE
Update all examples that start with "a" to use jest

### PR DESCRIPTION
**Pull request by bdemann**

### CODE REVIEW

File: examples/async_await/test/tests.ts [hunk 1]
```diff
@@ -1,54 +1,33 @@
...
+            expect(result).toHaveLength(32);
+        });
+
+        it('gets randomness even when there is multiple levels of awaits', async () => {
+            const result = await async_await_canister.getRandomnessIndirectly();
+
+            expect(result).toHaveLength(32);
+        });
+
+        it('gets randomness even when there are multiple awaits in multiple places at multiple levels', async () => {
```
Comment (bdemann) [inline examples/async_await/test/tests.ts:20]: "even where there are multiple levels of awaits" -@lastmjs

File: examples/async_await/test/tests.ts [hunk 2]
```diff
@@ -1,54 +1,33 @@
...
-    ];
+export function getTests(async_await_canister: ActorSubclass<_SERVICE>): Test {
+    return () => {
+        it('gets randomness when awaited once', async () => {
+            const result = await async_await_canister.getRandomnessDirectly();
+
+            expect(result).toHaveLength(32);
+        });
+
+        it('gets randomness even when there are multiple levels of awaits', async () => {
```
Comment (bdemann) [inline examples/async_await/test/tests.ts:14]: "even where there are multiple levels of awaits" -@lastmjs

File: examples/apollo_server/test/tests.ts [hunk 3]
```diff
@@ -1,74 +1,59 @@
...
+                data: {
+                    books: [
+                        { id: '0', title: 'The Awakening' },
+                        { id: '1', title: 'City of Glass' }
+                    ]
+                }
+            };
+            const response = await fetch(origin, {
+                method: 'POST',
+                headers: [['Content-Type', 'application/json']],
```
Comment (lastmjs) [inline examples/apollo_server/test/tests.ts:19]: Add X-Ic-Force_query to all GraphQL queries

File: examples/audio_and_video/test/tests.ts [hunk 4]
```diff
@@ -1,155 +1,101 @@
-import * as dns from 'node:dns';
-dns.setDefaultResultOrder('ipv4first');
-
-import { Test } from 'azle/test';
+import { expect, it, Test } from 'azle/test/jest';
```
Comment (lastmjs) [inline examples/audio_and_video/test/tests.ts:1]: Change Test to something like TestGetter

**Review by lastmjs — CHANGES_REQUESTED** [review]

**Review by lastmjs — DISMISSED** [review]

**Pull request merged**